trainer/lib/trainer/legacy_xcresult.rb

```diff
@@ -0,0 +1,586 @@
+require 'open3'
+
+require_relative 'xcresult/helper'
+
+module Trainer
+  module LegacyXCResult
+    # Model attributes and relationships taken from running the following command:
+    # xcrun xcresulttool formatDescription
```

Comment:
```suggestion
    # xcrun xcresulttool formatDescription --legacy
```

Not major since the name of the module is now `LegacyXCResult` so it's possible the `--legacy` flag would be inferred, but just in case anybody was trying to run this command and didn't make the inference.

Reply:
Nice catch!